src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -117,4 +117,15 @@ object IndexConstants {
   // To provide multiple paths in the globbing pattern, separate them with commas, e.g.
   // "/temp/1/*, /temp/2/*"
   val GLOBBING_PATTERN_KEY = "spark.hyperspace.source.globbingPattern"
+
+  /**
+   * Target minimum number of records per index data file for data skipping
+   * indexes.
+   *
+   * When index data is stored in files, the number of files to be written is
+   * limited to # of records / this value.
+   */
+  val DATASKIPPING_MIN_RECORDS_PER_INDEX_DATA_FILE =
+    "spark.hyperspace.index.dataskipping.minRecordsPerIndexDataFile"
+  val DATASKIPPING_MIN_RECORDS_PER_INDEX_DATA_FILE_DEFAULT = "100000"
```

Comment:
Is there any reason why use 100000 as default?

Reply:
The optimal value would vary depending on data skipping indexes, but we need a default value. Anything bigger than 1 would fit for the purpose. 100000 is chosen so that typical data skipping indexes with MinMaxSketches take at least >1MB per file.